utils/mkdocs_tx_commit.sh

```diff
@@ -17,14 +17,11 @@ if [[ $(git diff --exit-code mkdocs.yml) ]]; then
     git commit -m "Update mkdocs.yml translation" --no-verify
     git push
   else
-    # on push/workflow_dispatch create a pull request
+    # on push/workflow_dispatch directly push to branch
     git checkout ${GITHUB_REF_NAME}
-    BRANCH="update-mkdocs-tx-$RANDOM"
-    git checkout -b ${BRANCH}
     git add mkdocs.yml
     git commit -m "Update mkdocs.yml translation" --no-verify
```

Comment:
Unrelated, but why do we say `--no-verify` here? If there is a pre-commit hook, it will make it fail on `master`.

Reply:
pre-commit is run before (and I had issues with adding an extra line or removing, I don't recall exactly)